src/sentry/sentry_metrics/kafka_metrics_interface.py

```diff
@@ -0,0 +1,142 @@
+from concurrent.futures import Future
+from datetime import datetime
+from typing import Any, Mapping, Optional, Sequence, Union
+
+from arroyo import Message, Topic
+from arroyo.backends.kafka import KafkaPayload, KafkaProducer, build_kafka_configuration
+from django.conf import settings
+
+from sentry.sentry_metrics.metrics_interface import GenericMetricsBackend
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+from sentry.utils import json
+from sentry.utils.kafka_config import get_kafka_producer_cluster_options
+
+
+def build_mri(metric_name: str, type: str, use_case_id: UseCaseID, unit: Optional[str]) -> str:
+    mri_unit = "none" if unit is None else unit
```

Comment:
What does it mean for an mri_unit to be none? Under what circumstances does this happen and can it be avoided?

Reply:
Haven't gotten a chance yet but will look into how it works for Relay/envelopes currently. I may keep it depending on the way things work right now.